test/server/ServerSpec.js

```diff
@@ -60,20 +60,111 @@ describe('Server-side Unit test', function () {
 
   describe('Controller Test', function () {
     describe('item controller', function () {
-      it('should have getAll() method', function () {
-        expect(item.getAll).to.be.a('function');
+      describe('getAll() method', function () {
+        it('should be a function', function(){
+          expect(item.getAll).to.be.a('function');
+        });
+        it('should invoke the callback with items upon success', function(done) {
+          findAllStub.resolves(expectedArray)
```

Comment:
findAllStub is stub that I created that is stubbing User.findAll() not Item.findAll().
To make the test code for item controller work, create stub for Item class :)